util/tooltool/tooltool.py

```diff
@@ -833,39 +836,39 @@ def upload(manifest, message, base_urls, auth_file, region):
     # Upload the files, each in a thread.  This allows us to start all of the
     # uploads before any of the URLs expire.
     threads = {}
-    for filename, file in files.iteritems():
-        if 'put_url' in file:
-            log.info("%s: starting upload", filename)
+    for filename, file_ in files.iteritems():
+        if 'put_url' in file_:
+            log.info("%s: starting upload", (filename,))
```

Comment:
Note to self on Python 2.7:

```
>>> import logging
>>> logging.basicConfig(level=logging.INFO)
>>> logger = logging.getLogger(__name__)
>>> logger.setLevel(logging.INFO)
>>> logger.info("%s", 123)
INFO:__main__:123
>>> logger.info("%s", (123,))
INFO:__main__:(123,)
>>> logger.info("%s", ((123,),))
INFO:__main__:((123,),)
>>> logger.info("%s" % 123)
INFO:__main__:123
>>> logger.info("%s" % (123,))
INFO:__main__:123
>>> logger.info("%s" % ((123,),))
INFO:__main__:(123,)
>>> from __future__ import print_function
>>> print("%s", 123)
%s 123
>>> print("%s" % 123)
123
>>> print("%s" % (123,))
123
>>> print("%s" % ((123,),))
(123,)
>>> 
```

`%` treats `123` and `(123,)` similarly in both `logging` and `print`, but `,` is different.